pkg/adapter/nimbus-kyverno/processor/kcpbuilder.go

```diff
@@ -57,60 +57,79 @@ var nsBlackList = []string{"kube-system"}
 
 func clusterCocoRuntimeAddition(cnp *v1alpha1.ClusterNimbusPolicy, rule v1alpha1.Rule) kyvernov1.ClusterPolicy {
```

Comment:
Is a single policy sufficient ? Shouldn't we be having mutateExisting, and mutateNew ?

Reply:
the work is in progress ClusterScoped mutation policy generation logic @shivaccuknox as we discussed earlier.

Reply:
kcp is WIP